keepalived-vip/main.go

```diff
@@ -39,15 +40,22 @@ var (
 	useUnicast = flags.Bool("use-unicast", false, `use unicast instead of multicast for communication
 		with other keepalived instances`)
 
-	password = flags.String("vrrp-password", "", `If set it will use it as keepalived password instead of the 
+	password = flags.String("vrrp-password", "", `If set it will use it as keepalived password instead of the
 		generated one using information about the nodes`)
 
+	enableIPVSConntrack = flags.Bool("enable-ipvs-conntrack", true,
+		`enable ipvs conntrack in net/ipv4/vs/conntrack`)
+
 	// sysctl changes required by keepalived
 	sysctlAdjustments = map[string]int{
 		// allows processes to bind() to non-local IP addresses
 		"net/ipv4/ip_nonlocal_bind": 1,
-		// enable connection tracking for LVS connections
-		"net/ipv4/vs/conntrack": 1,
```

Comment:
This is not optional. Without connection tracking is not possible to receive the response from the backend. 
With the change in network addresses is not possible to use DR.